tests/angular_devkit/build_angular/hello-world-app-ve/src/test.ts

```diff
@@ -11,13 +11,16 @@ import 'zone.js/dist/zone-testing';
 import { getTestBed } from '@angular/core/testing';
 import {
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting,
+  platformBrowserDynamicTesting
```

Comment:
A TSLint rule would be useful here, in order to stick to stick to a partern and avoid removing and adding the trailing comma between MRs.

Reply:
These are tests assets which are not linted, since they may differ from the actual code patterns of the repo.